.github/workflows/test.yml

```
@@ -22,7 +22,11 @@ jobs:
           ${{ runner.os }}-
     - name: Installation des dépendances
       run: pip install -r requirements.txt
-    - name: Vérification du schéma et des fichiers d'exemples
+    - name: Vérification du schéma statique et des fichiers d'exemples associés
       run: |
-        frictionless validate --type schema schema.json
-        frictionless validate --schema schema.json exemple-valide.csv
\ No newline at end of file
+        frictionless validate --type schema statique/schema-statique.json
+        frictionless validate --schema statique/schema-statique.json statique/exemple-valide-statique.csv
```

```suggestion
        frictionless validate datapackage.json
```
Instead of validate each examples, you can validate directly the file datapackage.json which will check them.